spec/client_spec.js

```diff
@@ -148,7 +148,7 @@ describe('Client', () => {
       })
     })
 
-    it('throws when domain is invalid', () => {
+    it('clear iframe and append error msg to body when domain is invalid', () => {
```

Comment:
nit re proper use of the imperative: `clear` -> `clears`, `append` -> `appends`

Reply:
haha yess my grammar is the worst :P